controllers/collection.lua

```diff
@@ -87,53 +78,106 @@ CollectionController.GET.user_collections = function (self)
     end
 end
 
-CollectionController.GET.collection = function (self)
+CollectionController.GET.collections = function (self)
```

Comment:
Actually, this returns a single collection. I think the old singular name was better.